distribution/docs/src/main/resources/_contents/_formats/file-types-supported-contents.adoc

```diff
@@ -0,0 +1,1544 @@
+==== Supported File Formats
+
+Using the various input transformers, ${branding} supports ingest of the following MIME types.
+
+.[[supported_file_types]]Supported File Types
```

Comment:
See https://github.com/codice/ddf/pull/1449. This list is what you can ingest without crashing. It's been moved to an appendix. The list of types that metadata can be extracted from is much smaller.